src/lc0ctl/describenet.cc

```diff
@@ -129,28 +143,67 @@ void ShowNetworkWeightsInfo(const pblczero::Net& weights) {
   COUT << "\nWeights";
   COUT << "~~~~~~~";
   const auto& w = weights.weights();
-  COUT << Justify("Blocks") << w.residual_size();
-  int se_count = 0;
-  for (size_t i = 0; i < w.residual_size(); i++) {
-    if (w.residual(i).has_se()) se_count++;
-  }
-  if (se_count > 0) {
-    COUT << Justify("SE blocks") << se_count;
-  }
-
-  COUT << Justify("Filters")
-       << w.input().weights().params().size() / 2 / 112 / 9;
-  COUT << Justify("Policy") << (w.has_policy1() ? "Convolution" : "Dense");
-  if (!w.has_policy1()) {
-    int policy_channels = w.policy().biases().params().size() / 2;
-    if (policy_channels == 0) {
-      policy_channels = w.policy().bn_means().params().size() / 2;
+
+  const auto attn_body = w.encoder_size() > 0;
+  if (attn_body) {
```

Comment:
Optional nit: this function is becoming long, and it's easy to split it, maybe do that?

Reply:
Done